core/validation/initial_check.py

```diff
@@ -63,6 +63,15 @@ def extract_submission_details(
     if invalid_sheets:
         return {"Invalid Sheets": missing_sheets}
 
+    # if None, then user is submitting via API
```

Comment:
I wonder if rather than relying on this implicit check maybe we can set a user agent string or other header, to indicate the request is coming from the frontend?

Reply:
The data-store doesn't actually care if it's coming from the front-end. This is just implied because the front-end will never send a request to data-store with no place names because the users must be authorized to submit if via the frontend. Unless you think this is worth the extra effort right now I'd be tempted to get this in and tested ASAP and add this to the backlog.